dev-libs/libcdio/libcdio-2.0.0.recipe

```diff
@@ -0,0 +1,120 @@
+SUMMARY="A library to encapsulate CD-ROM reading and control"
+DESCRIPTION="The libcdio package contains a library for CD-ROM and CD image \
+access. Applications wishing to be oblivious of the OS- and device-dependent \
+properties of a CD-ROM or of the specific details of various CD-image formats \
+may benefit from using this library."
+HOMEPAGE="https://www.gnu.org/software/libcdio/"
+COPYRIGHT="2002-2017 Rocky Bernstein and Herbert Valerio Riedel"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://ftpmirror.gnu.org/libcdio/libcdio-$portVersion.tar.gz
+	https://ftp.gnu.org/gnu/libcdio/libcdio-$portVersion.tar.gz"
+CHECKSUM_SHA256="1b481b5da009bea31db875805665974e2fc568e2b2afa516f4036733657cf958"
+PATCHES="libcdio-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+libcdioLibs="libcdio libcdioxx libiso9660"
+libVersion="0.0.0"
+libcdioVersion="18.0.0"
+libcdioxxVersion="1.0.0"
+libiso9660Version="11.0.0"
+for i in lib $libcdioLibs; do
+	eval "${i}VersionCompat=\"\$${i}Version compat >= \${${i}Version%%.*}\""
+done
+otherlibcdioLibs="libiso9660++ libudf"
```

Comment:
Maybe we should not assume that `libiso9660++.so.0.0.0` and `libudf.so.0.0` will always end with `.0.0.0`.
So, what do you think of this little variant?
```diff
-libcdioLibs="libcdio libcdioxx libiso9660"
+libcdioLibs="
+	libcdio
+	libcdioxx
+	libiso9660
+	libiso9660xx
+	libudf
+	"
-libVersion="0.0.0"
-libcdioVersion="18.0.0"
+libcdioVersion=18.0.0
-libcdioxxVersion="1.0.0"
+libcdioxxVersion=1.0.0
-libiso9660Version="11.0.0"
+libiso9660Version=11.0.0
+libiso9660xxVersion=0.0.0
+libudfVersion=0.0.0
 for i in lib $libcdioLibs; do
 	eval "${i}VersionCompat=\"\$${i}Version compat >= \${${i}Version%%.*}\""
 done
-otherlibcdioLibs="libiso9660++ libudf"
```

Of course, the other comments are part of this variant.
I'm also suggesting a few cosmetic/style changes, but please feel free to ignore them if you wish.

Reply:
@fbrosson looking at the previous version both libs didn't change libversion, hence why I wanted to pull them into a separate check, started out with the changes you mention here, but I'm ok if you push the changes to the suggested ones :)

Reply:
Cool, thanks!